test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -2114,14 +2112,15 @@ def create_temp_dir(self, hostname=None, suffix='', prefix='PtlPbs',
                 self.run_copy(hostname, src=tmpdir, dest=tmpdir,
                               recursive=True, uid=uid, gid=gid,
                               level=level)
-                self.chmod(hostname, path=tmpdir, mode=mode, runas=asuser)
             else:
                 # copy temp dir created on localhost to remote as current user
                 self.run_copy(hostname, src=tmpdir, dest=tmpdir,
-                              level=level, preserve_permission=True,
+                              level=level, preserve_permission=False,
```

Comment:
just wondering, if we are going to do a chmod anyways for either case, can we get rid of the if-else block and just call run_copy with preserve_permission=False? chmod should take care of fixing the permissions right?